.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -528,3 +534,9 @@ for (const compassDistribution of COMPASS_DISTRIBUTIONS) {
       - func: create_static_analysis_report
         vars:
           first_party_dependency_filenames: <%= firstPartyDepsFilenames.map(f => '.sbom/' + f).join(',') %>
+
+  - name: publish-e2e-tests
+    tags: ['run-on-pr']
```

Comment:
TODO: remove this before merging, turned it on for now just to test

Reply:
Can we do a "dry run" to confirm that it's working on PRs?